Sources/MapboxNavigation/LanesView.swift

```diff
@@ -59,7 +59,7 @@ open class LanesView: UIView, NavigationComponent {
             }
         }
         
-        guard !subviews.isEmpty && subviews.contains(where: { !$0.isValid }) else {
+        guard !subviews.isEmpty && subviews.contains(where: { $0.isValid }) else {
```

Comment:
Are we checking that "all subviews are valid" or "any subview is valid"?
```suggestion
        guard !subviews.isEmpty && !subviews.contains(where: { !$0.isValid }) else {
```

Reply:
We are checking that at least 1 subview is valid